### PR TITLE
Update indoors.json - Add Copy Room

### LINKED ISSUE #2131: Indoor: Add Copy Room
I would like to suggest adding 'Copy Room' as room type to the Indoor theme.
| name | ref | notes |
| - | - | - |
| Copy Room | [`room=printer`](https://taginfo.openstreetmap.org/tags/room=printer#overview) | [in use] A [(photo)copy(ing) room](https://en.wiktionary.org/wiki/copy_room) or 'printer room' is a room with a [(photo)copy machine and/or printer and/or scanner and/or similar device](https://wiki.openstreetmap.org/wiki/Tag:amenity=printer), sometimes with computer(s) intended mainly for this purpose. It may also provide various (white) paper, cutting and perforation machine, scissors, stamps, stickers, envelopes. could host 'file room' (on its own I suppose it is tagged as `room=storage`).

Found mainly in educational buildings, sometimes in library buildings but there printers are more often part of the library room.

Some do not have a door, so it is understandable if it is considered indoor area / corridor part, though `door=no` exists.

---

(sidenote: pieter can click the edit button on this post to see some other examples of room types. however, most are minor/ambiguous so I won't propose them in this issue.)

### PR DESCRIPTION
**Pull request by danieldegroot2**
Adds room=printer (without proper icon)

Fixes #2131

### CODE REVIEW

**Comment by pietervdvn** [comment]
Good first step! You've indicated in the json that you'd like an icon, but didn't include the icon file.

Either we drop that (simple) or we include the icon file and its copyright information.

File: assets/layers/indoors/indoors.json [hunk 1]
```diff
@@ -474,6 +476,13 @@
           },
           "icon": "./assets/layers/indoors/room_office.svg"
         },
+        {
+          "if": "room=printer",
+          "then": {
+            "en": "This is a copy room"
+          },
+          "icon": "./assets/layers/indoors/room_printer.svg"
```
Comment (pietervdvn) [inline assets/layers/indoors/indoors.json:484]: Either we remove this line and go ahead, or 'room_printer.svg' is added with copyright information.

Comment (danieldegroot2) [reply assets/layers/indoors/indoors.json]: Removed it for now.

**Review by pietervdvn — APPROVED** [review]

**Comment by pietervdvn** [comment]
Excellent! Feel free to propose more changes (as PR)

**Pull request merged**